doc/nrf/releases/release-notes-latest.rst

```diff
@@ -193,6 +193,11 @@ The following list summarizes the most important changes inherited from upstream
 
   * Merged a new work queue implementation.
     See `this comment <kwork API changes_>`_ for details on the API changes.
+  * :c:func:`k_sem_init` users should now provide :c:macro:`K_SEM_MAX_LIMIT` as the limit value of semaphores that do not have explicit maximum limits and are instead just used for counting.
+    This is meant as a replacement for using ``UINT_MAX``.
+  * The :option:`CONFIG_THREAD_MONITOR` and :option:`CONFIG_THREAD_NAME` options are no longer marked experimental.
+  * The deprecated ``k_mem_domain_destroy`` and ``k_mem_domain_remove_thread`` APIs have been removed.
+  * :c:func:`device_usable_check` and :c:func:`device_is_ready` can now be called from userspace.
```

Comment:
We need a common style for all entries here, and this is "Did XYZ.". So:

```suggestion
  * Added a :c:macro:`K_SEM_MAX_LIMIT` define that users should provide in :c:func:`k_sem_init` as the limit value of semaphores that do not have explicit maximum limits and are instead just used for counting.
    This is meant as a replacement for using ``UINT_MAX``.
  * Moved the :option:`CONFIG_THREAD_MONITOR` and :option:`CONFIG_THREAD_NAME` options from experimental to production quality.
  * Removed the deprecated ``k_mem_domain_destroy`` and ``k_mem_domain_remove_thread`` APIs.
  * Updated the :c:func:`device_usable_check` and :c:func:`device_is_ready` functions so that they can be called from user space.
```

Reply:
> We need a common style for all entries here, and this is "Did XYZ.". So:

applied